context/brand-context/HISTORY.md

```diff
@@ -1,5 +1,10 @@
 # History
 
+## 18.0.0 (2021-11-01)
```

Comment:
You have added the new version here but haven't updated the `package.json` so it won't publish